### ISSUE TITLE
observers & observe-parents params no available

### ISSUE THREAD

**Issue by b82** [ksachdeva/angular-swiper#12]
Hi,

i have a hidden slider that appears on click.
On web version this can be rightly done using observer="true" observe-parents="true", which are available for web apps.
Is it possible have those params also for this plugin?

Thank you and best regards

**Comment by brh55** [comment]
This could be accomplished quite easily, however, @ksachdeva, do you believe it may be best to assign default params for at least boolean values? It will prevent a lot of similar request in the future.

**Comment by ksachdeva** [comment]
Hi @brh55 since swiper has lot of options may be we should think about an approach using which we do not have to update the directive. One approach is to pass an object prepared by controller or json attribute on the element. This however destroys the 'semantic' aspect of the directive.  Look at the fourth option on this gist https://gist.github.com/CMCDragonkai/6282750

**Comment by brh55** [comment]
@ksachdeva Hmm, if i'm not mistaken, @b82 should be able to declare the param with the 

```
override-parameters="{ 'observer': true, 'observe-parents': true}"
```

If this is the case, I think the override-parameter attribute is sufficient to extend any additional params for swiper.

**Comment by ksachdeva** [comment]
This is correct. I have not tested override-parameters myself but seems to implement passing of generic attributes. One of the contributors added this.
